build/tasks/spec-task.coffee

```diff
@@ -104,6 +104,30 @@ module.exports = (grunt) ->
 
       callback(null, error)
 
+  runMainProcessSpecs = (callback) ->
+    appPath = getAppPath()
+    resourcePath = process.cwd()
+    mainProcessSpecsPath = path.resolve('spec/browser')
```

Comment:
is there a reason you chose to go with `spec/browser` rather than `spec/main`? IMO `main` is a bit more clear